### PR TITLE
SUMO-235493 Fix the CloudFormation template for Kinesis log sources

### PR DESCRIPTION
**Pull request by yuting-liu**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

We changed following SizeInMBs from `4` to `1` as the recommendations we gave to our customer for their Kinesis firehose configuration
```suggestion
        BufferingHints:
          IntervalInSeconds: 60
          SizeInMBs: 1
```

I have no access to the old link reference thus I added a new yaml file directly in this folder. @kimsauce Please let me know if this works.

- [X] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request needs to update the downloadable CloudFormation template for CloudWatch logs ingestion via Kinesis log source.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

File: static/files/KinesisFirehoseCWLogs.template.yaml [hunk 1]
```diff
@@ -0,0 +1,348 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "Template to setup the AWS CloudWatch Logs collection using Amazon Kinesis Firehose and send the data to provided Sumo Logic URL."
```
Comment (kimsauce) [inline static/files/KinesisFirehoseCWLogs.template.yaml:2]:
```suggestion
Description: "Template to set up the AWS CloudWatch Logs collection using Amazon Kinesis Firehose and send the data to provided Sumo Logic URL."
```

**Comment by kimsauce** [comment]
thank you, @yuting-liu !

**Comment by yuting-liu** [comment]
@kimsauce Can you please merge the PRas I don't have the permission to commit the change? Thank you.

**Comment by kimsauce** [comment]
> @kimsauce Can you please merge the PRas I don't have the permission to commit the change? Thank you.

Sure thing!

**Pull request merged**